base/database/database.go

```diff
@@ -36,11 +36,8 @@ func OnConflictDoUpdateExpr(db *gorm.DB, keys []string, updateExprs ...UpExpr) *
 	for _, key := range keys {
 		confilctColumns = append(confilctColumns, clause.Column{Name: key})
 	}
-	if updateColsValues != nil {
-		return db.Clauses(clause.OnConflict{
-			Columns:   confilctColumns,
-			DoUpdates: clause.Assignments(updateColsValues),
-		})
-	}
-	return db
+	return db.Clauses(clause.OnConflict{
+		Columns:   confilctColumns,
+		DoUpdates: clause.Assignments(updateColsValues),
+	})
```

Comment:
I understand `updateColsValues` is never `nil` beacuse it's created as an empty map.
But should not the condition be changed to `len(updateColsValues) > 0` to keep the original logic/intent?